cmd/prom2json/main.go

```diff
@@ -19,10 +19,8 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/prometheus/common/log"
-
 	dto "github.com/prometheus/client_model/go"
-
+	"github.com/prometheus/common/log"
```

Comment:
Please don't re-arrange the import lines. The grouping in blocks follows the general Prometheus style.

(stdlib, then unnamed from other repos, then named from other repos, then unnamed from same repo, then named from same repo)